lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -503,6 +503,14 @@ void EventThreadFunction() {
             SendContinuedEvent();
             break;
           case lldb::eStateExited:
+            const int exit_status = process.GetExitStatus();
+            const char *const exit_description = process.GetExitDescription();
+            g_dap.SendFormattedOutput(
+                OutputType::Console,
+                "Process %" PRIu64 " exited with status = %i (0x%8.8x) %s\n",
+                process.GetProcessID(), exit_status, exit_status,
+                exit_description ? exit_description : "");
+
```

Comment:
This code seems to be a duplication of `Process::GetStatus`. Maybe we should expose `Process::GetStatus` in SBPorcess to ensure lldb-dap and lldb output are not out of sync if future modification is added to `Process::GetStatus`.